lib/recipe_box_web/controllers/cuisine_controller.ex

```diff
@@ -0,0 +1,24 @@
+defmodule RecipeBoxWeb.CuisineController do
+  use RecipeBoxWeb, :controller
+
+  alias RecipeBox.Cuisine
+  
+  @doc """
+  The conn is the Plug connection. 
+  The Plug takes a connection and returns a connection,
+  and the conn is a struct that
+  contains all the information (body, cookies, headers ...).
+  """
+  def new(conn, params) do
+    changeset = Cuisine.changeset(%Cuisine{}, params)
+    render(conn, "new.html", changeset: changeset)
+  end
+
+  def create(conn, %{"cuisine" => cuisine_params}) do
+    {:ok, cuisine} = Cuisine.create_cuisine(cuisine_params)
```

Comment:
This pattern match means that if `create_cuisine/1` returns `{:error, changeset}`, we'll get a `500` in the client. I'm not looking at the Trello board, so maybe presenting errors is another story, in which case please disregard this comment.